straxen/plugins/pulse_processing.py

```diff
@@ -250,18 +257,22 @@ def compute(self, raw_records_he, start, end):
         return dict(records_he=result['records'],
                     pulse_counts_he=result['pulse_counts'])
 
+
 ##
 # Software HE Veto
 ##
 
+# Reproducible random state for HE Veto function
+prng = np.random.RandomState(123)
```

Comment:
Does it make sense to put this inside the function or did you put it here on purpose? Otherwise when you do straxen.software_he_veto(records) you still have a different result than from the processing. 

What do you think?